src/App.tsx

```diff
@@ -24,37 +28,19 @@ export const App: React.FC = () => (
     <h1 className="App__title">Static list of todos</h1>
 
     <section className="TodoList">
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">HTML</h2>
-
-        <a className="UserInfo" href="mailto:Sincere@april.biz">
-          Leanne Graham
-        </a>
-      </article>
-
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">CSS</h2>
-
-        <a className="UserInfo" href="mailto:Sincere@april.biz">
-          Leanne Graham
-        </a>
-      </article>
-
-      <article className="TodoInfo TodoInfo--completed">
-        <h2 className="TodoInfo__title">JS</h2>
-
-        <a className="UserInfo" href="mailto:Shanna@melissa.tv">
-          Ervin Howell
-        </a>
-      </article>
-
-      <article className="TodoInfo">
-        <h2 className="TodoInfo__title">React</h2>
-
-        <a className="UserInfo" href="mailto:Nathan@yesenia.net">
-          Clementine Bauch
-        </a>
-      </article>
+      {todos.map(todo => (
+        <article className={classNames(
+          'TodoInfo',
+          {
+            'TodoInfo--completed': todo.completed,
+          },
+        )}
+        >
+          <h2 className="TodoInfo__title">{todo.title}</h2>
+
+          {todo.user && <UserInfo user={todo.user} />}
+        </article>
```

Comment:
Don't forget about key attribute
```suggestion
        <article 
          className={cn('TodoInfo',
            'TodoInfo--completed': todo.completed,
          })}
          key={todo.id}
        >
          <h2 className="TodoInfo__title">{todo.title}</h2>

          {todo.user && <UserInfo user={todo.user} />}
        </article>
```